reporter/reporter.py

```diff
@@ -115,10 +121,81 @@ def notify():
     with CrateTranslator(DB_HOST, DB_PORT, DB_NAME) as trans:
         trans.insert([payload])
 
-    msg = "Notification successfully processed"
+    msg = 'Notification successfully processed'
     logger.info(msg)
     return msg
 
 
-if __name__ == '__main__':
-    app.run(host=LOCAL, port=8668)
+def query_1T1E1A(attrName, entityId, type=None):
```

Comment:
no filters?

Reply:
let's see if I can include any in this same PR. Any in particular you had in mind?

Reply:
i am not thinking about their implementation, but it would be better if the method already foresee them.

Reply:
Sure, I was not concerned because the definite list of params was still to be defined, adding them is trivial though better at the end. Added params from the current definition and will update accordingly :)